dvc/commands/experiments/__init__.py

```diff
@@ -58,6 +58,16 @@ def add_parser(subparsers, parent_parser):
     hide_subparsers_from_help(experiments_subparsers)
 
 
+def add_keep_selection_flag(experiments_subcmd_parser):
+    experiments_subcmd_parser.add_argument(
+        "--keep",
+        action="store_true",
+        default=False,
+        help="Keep the selected experiments instead of removing them "
+        "(use it with `--rev`' and `--num` or with experiment names).",
```

Comment:
does it work with all available options? (it seems that `(use it with `--rev`' and `--num` or with experiment names).` is not needed)

Reply:
Indeed.